src/main/java/jarvis/command/FindCommand.java

```diff
@@ -0,0 +1,50 @@
+package jarvis.command;
+
+import java.util.List;
+
+import jarvis.JarvisException;
+import jarvis.storage.Storage;
+import jarvis.task.Task;
+import jarvis.task.TaskList;
+
+/**
+ * FindCommand --- find tasks.
+ */
+public class FindCommand extends Command {
+    /**
+     * Constructor.
+     *
+     * @param command the command entered by the user.
+     */
+    public FindCommand(String command) {
+        super(command);
+    }
+
+    /**
+     * Executes the command.
+     *
+     * @param tasks the list of tasks.
+     * @param storage stores the tasks locally.
+     * @return response after executing the command.
+     * @throws JarvisException exception for invalid commands.
+     */
+    @Override
+    public String execute(TaskList tasks, Storage storage) throws JarvisException {
+        List<Task> found = tasks.findTasks(super.getDescription());
```

Comment:
Perhaps you could rename your List to specify the type of items found?
```suggestion
        List<Task> tasksFound = tasks.findTasks(super.getDescription());
````